### PR TITLE
Assurance 2.1.1 release notes

### PR DESCRIPTION
**Pull request by prudrabhat**
## Description
Add release noted for Assurance 2.1.1

## Related Issue
N/A

## Motivation and Context
Assurance 2.1.1 has been released. Add release notes to docs.

## How Has This Been Tested?
N/A

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by calebk1 — APPROVED** [review]
LGTM!

Does the iOS Assurance not have the same updates?

**Comment by prudrabhat** [comment]
> LGTM!
> 
> Does the iOS Assurance not have the same updates?

iOS Updates were already released and documented a [few weeks back](https://developer.adobe.com/client-sdks/documentation/platform-assurance-sdk/release-notes/#ios-assurance-312)!.

File: src/pages/documentation/release-notes/index.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## July 21, 2023
+
+### Android Assurance 2.1.1
+
+- Added support for chained events.
+- Improvements to the test app.
```
Comment (emdobrin) [inline src/pages/documentation/release-notes/index.md:12]: We usually don't include non-sdk related notes in this repo, but we do in github tags. Is this a change requested by one of our customers or is there a strong reason for including it?

Comment (prudrabhat) [reply src/pages/documentation/release-notes/index.md:12]: I wasn't aware of that non-sdk changes can be omitted. I will remove them 👍🏼

File: src/pages/documentation/platform-assurance-sdk/release-notes.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 Release notes and change logs for the Adobe Experience Platform Assurance extension
 
+## July 21, 2023
```
Comment (emdobrin) [inline src/pages/documentation/platform-assurance-sdk/release-notes.md:5]: was this released on July 20?

Comment (prudrabhat) [reply src/pages/documentation/platform-assurance-sdk/release-notes.md]: Yes! You are correct. Updated it now.

**Review by emdobrin — APPROVED** [review]

**Pull request merged**